components/client.jsx

```diff
@@ -47,7 +47,8 @@ export default class Client extends Component{
         super(props);
         this.state = {
             room: null,
-            theme: props.theme === undefined ? 'dark' : props.theme,  // Client theme (dark/light)
+            theme: props.theme === undefined ? 'dark' : 
```

Comment:
IMO if you're nesting ?:'s then it's probably time to  move the logic out, but this is ok ;)

Reply:
Yep, I would clean the code soon.